Api/Web.config

```diff
@@ -4,13 +4,16 @@
   http://go.microsoft.com/fwlink/?LinkId=169433
   -->
 <configuration>
+  <appSettings>
+    <add key="okta:ClientId" value="lXSenLxxPi8kQVjJE5s4" />
```

Comment:
I think we should avoid putting real values here.

Maybe we replace the `TenantUrl` with `https://dev-xyz.okta.preview.com`